kotlinx-coroutines-core/concurrent/test/sync/SemaphoreStressTest.kt

```diff
@@ -41,12 +47,38 @@ class SemaphoreStressTest : TestBase() {
     }
 
     @Test
-    fun stressCancellation() = runBlocking(Dispatchers.Default) {
-        val n = 10_000 * stressTestMultiplier
+    fun testNativeSpecificStress() = runMtTest {
+        /*
+         * Dispatchers.Default is still single-threaded, so here we're doing actual concurrency
```

Comment:
Could you explain this comment?

Reply:
This is a leftover from times `Dispatchers.Default` was backed by a single worker